src/ci/resources/papi_v2_application.conf

```diff
@@ -12,7 +12,7 @@ services {
       # The carbonite section contains carbonite-specific options
       carbonite-metadata-service {
         # enable carboniting
-        enabled = false
+        enabled = true
```

Comment:
This enables Carboniting for all Centaur tests no? Do we want to check if the carboniting was successful for all the existing tests or just specific test cases?

Reply:
It turns on carboniting for *all* tests. We assert that it was successful on the `workflowSuccess` and `workflowFailure` test types.

Are you saying I might want to add the verification in on the other more specialist test types too?

Reply:
Hmm I would want to know Carboniting worked for all tests, but don't we also want to know that we didn't break non-Carbonited metadata too?